small-talk/util/smolCwypto.test.js

```diff
@@ -1,25 +1,24 @@
 import { createCookieHash } from "@/util/smolCwypto.js";
-
 describe('createCookieHash', () => {
-    it('should create a valid hash', () => {
-        const data = { username: 'user123', timestamp: Date.now() };
-        const secretKey = 'mySecretKey';
+    const secretKey = 'mySecretKey';
 
+    it('should create a valid hash', () => {
+        const data = { username: 'testUser',  password: 'testPassword', timestamp: Date.now() };
         const hash = createCookieHash(data, secretKey);
 
         // Add your assertions here
-        expect(hash).to.be.a('string');
-        expect(hash).to.have.lengthOf(64); // SHA-256 hash length
+        expect(typeof hash).toBe('string');
+        //expect(hash).to.have.lengthOf(64); // SHA-256 hash length
```

Comment:
What's the reason for not checking length anymore?

Reply:
It's because the security team is now using a hashing algorithm which doesn't always produce a length of 64.